src/apps/reservation-list/list/reservation-list.dev.tsx

```diff
@@ -224,14 +224,6 @@ export default {
       defaultValue: "Close pause reservations modal",
       control: { type: "text" }
     },
-    dateInputsStartDateLabelText: {
```

Comment:
If these texts are no longer used at all, I would suggest removing them from the CMS as well :)